phpstan.neon

```diff
@@ -0,0 +1,29 @@
+includes:
+    - vendor/timeweb/phpstan-enum/extension.neon
+    - vendor/szepeviktor/phpstan-wordpress/extension.neon
+
+parameters:
+    level: 0
+    paths:
+        - src/
+    bootstrapFiles:
+        - give.php
+    scanDirectories:
+        - includes/
+    excludePaths:
+        analyse:
+            - src/LegacySubscriptions/includes/ # Scan but do not analyse
+        analyseAndScan:
+            - src/Form/LegacyConsumer/ServiceProvider.php # Needs a closer look
+            - src/Framework/FieldsAPI/Concerns/HasType.php
+            - src/Framework/Migrations/
+            - src/Framework/PaymentGateways/
```

Comment:
Why are we skipping theses?

Reply:
@JasonTheAdams these are ignored as the required changes are larger refactors than I wanted to include in this PR.

See [32132d6](https://github.com/impress-org/givewp/pull/6513/commits/32132d6ba5a852bbf61ec6f65cae536345616013).

Reply:
That's fair. Let's just make sure we chip away at these in future PRs until they're all done. 😃